src/connector.rs

```diff
@@ -612,4 +630,33 @@ impl EthConnectorContract {
     fn check_used_event(&self, key: &str) -> bool {
         sdk::storage_has_key(&self.used_event_key(key))
     }
+
+    /// Get Eth connector paused flags
+    pub fn get_paused_flags(&self) {
+        let data = self.get_paused().try_to_vec().expect(ERR_FAILED_PARSE);
```

Comment:
Seems like an abuse of `ERR_FAILED_PARSE` since we normally use it on input args, not output data I think. But it doesn't really matter because this will never fail anyways.